spec.html

```diff
@@ -24710,7 +24706,7 @@
 
       <!-- es6num="20.2.2.18" -->
       <emu-clause id="sec-math.hypot">
-        <h1>Math.hypot ( _value1_ , _value2_ , &hellip;_values_ )</h1>
+        <h1>Math.hypot ( _value1_ , _value2_ , ..._values_ )</h1>
         <p>`Math.hypot` returns an implementation-dependent approximation of the square root of the sum of squares of its arguments.</p>
```

Comment:
We have total control over the syntax used to describe semantics in the spec. It's just an editorial convention. We can use emoji if we like :)